includes/models/model.llms.order.php

```diff
@@ -632,13 +641,37 @@ public function get_customer_name() {
 	}
 
 	/**
-	 * An array of default arguments to pass to $this->create()
-	 * when creating a new post
+	 * Retrieve the customer's full billing address
+	 *
+	 * @since [version]
+	 *
+	 * @return string
+	 */
+	public function get_customer_full_address() {
```

Comment:
We should get a test for this function and should probably also consider (later) where else addresses are output.

I know there's other places that addresses are output.

I'm also wondering if we should consider outputting these with newlines (for proper actual address formatting) and then there's the i18n consideration that this is the US address format and other countries have their own formats..

Reply:
@thomasplevy 
I've added a test for this function:
https://github.com/gocodebox/lifterlms/blob/e626c2bdaf3250d49b24129fb24f119357082ad9/tests/phpunit/unit-tests/models/class-llms-test-model-llms-order.php#L1266

Ok the only purpose for this was to reduce redundancy.
Add a todo here and open an issue or do you want to address it in this PR?
In this case I should document myself on i18n addresses :D